src/additional/dateMultipleFormat.js

```diff
@@ -60,10 +71,10 @@
 		dateFormat = dateFormat.split( separator );
 
 		var now = new Date(),
-			year  = value[ dateFormat.indexOf( "YYYY" ) ] ||
-			( ( Math.ceil( now.getFullYear() / 100 ) - 1 ) + value[ dateFormat.indexOf( "YY" ) ] ),
-			month = value[ dateFormat.indexOf( "MM" ) ] || value[ dateFormat.indexOf( "M" ) ],
-			day   = value[ dateFormat.indexOf( "DD" ) ] || value[ dateFormat.indexOf( "D" ) ];
+			year        = value[ $.inArray("YYYY",dateFormat) ] ||
+				( ( Math.ceil( now.getFullYear() / 100 ) - 1 ) + value[ $.inArray("YY",dateFormat) ] );
```

Comment:
If year is just 2 characters, for example `15`
So this code 
```js
( Math.ceil( now.getFullYear() / 100 ) - 1 ) + value[ $.inArray("YY",dateFormat) ]
```
will return `2015` and it's a string